app/models/project.rb

```diff
@@ -1,3 +1,6 @@
 class Project < ApplicationRecord
+  has_many :tasks, dependent: :destroy
+
+  validates_associated :tasks
```

Comment:
This validation already has when has a `has_many` association

```suggestion
```

Reply:
When with your validator: (task is invalid)
<img width="705" alt="image" src="https://github.com/fs/task_tracker_itis_2023/assets/78980031/61f998d3-0663-4b07-a2b5-d4216ab5d476">

When without your validator: (task is invalid)

<img width="724" alt="image" src="https://github.com/fs/task_tracker_itis_2023/assets/78980031/19692be8-aa3e-4496-8d18-8ecc3b7399fc">